ts_src/psbt.ts

```diff
@@ -1717,3 +1738,38 @@ function randomBytes(options?: RngOpts): Buffer {
   const rng = options.rng || _randomBytes;
   return rng(32);
 }
+
+export interface UnblindWitnessUtxoResult {
+  value: string;
+  ag: Buffer;
+  abf: Buffer;
+  vbf: Buffer;
+}
+
+export function unblindWitnessUtxo(
+  prevout: WitnessUtxo,
+  blindingPrivKey: Buffer,
+): UnblindWitnessUtxoResult {
+  const unblindPrevout: UnblindWitnessUtxoResult = {
+    value: '',
+    ag: Buffer.alloc(0),
+    abf: Buffer.alloc(0),
+    vbf: Buffer.alloc(0),
+  };
+
+  const unblindProof = confidential.unblindOutput(
+    prevout.nonce,
+    blindingPrivKey,
+    prevout.rangeProof!,
+    prevout.value,
+    prevout.asset,
+    prevout.script,
+  );
+
+  unblindPrevout.ag = unblindProof.asset;
+  unblindPrevout.value = unblindProof.value;
+  unblindPrevout.abf = unblindProof.assetBlindingFactor;
+  unblindPrevout.vbf = unblindProof.valueBlindingFactor;
+
+  return unblindPrevout;
```

Comment:
this could be simplified:
```suggestion
  const unblindProof = confidential.unblindOutput(
    prevout.nonce,
    blindingPrivKey,
    prevout.rangeProof!,
    prevout.value,
    prevout.asset,
    prevout.script,
  );

  return {
    value: unblindProof.value,
    ag: unblindProof.asset,
    abf: unblindProof.assetBlindingFactor,
    vbf: unblindProof.valueBlindingFactor,
  };
```